HazelcastCloudTests/csharphazelcastcloudtests/Remote/IRemoteControllerClient.cs

```diff
@@ -0,0 +1,204 @@
+﻿// Copyright (c) 2008-2021, Hazelcast, Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Hazelcast.Testing.Remote
+{
+    /// <summary>
+    /// Defines a remote controller client.
+    /// </summary>
+    public interface IRemoteControllerClient
```

Comment:
Wondering... aren't all those files duplicating what we already have in our Testing project, and could we reuse them?

Reply:
Feeling same, the whole `Remote` directory is edited and replaced under `Hazelcast.Testing. I think whole cloud think can be done int the main repository. And related workflow can be controlled here. Having two different code base makes things difficult. However, I prefer to done everything together instead piece by piece. So,  we can wait the QA for that. WDYT?

Reply:
totally fine.